client/src/featureform/format.py

```diff
@@ -4,7 +4,10 @@
 divider = "-----------------------------------------------"
 
 def format_rows(format_obj, format_obj_2=None, format_obj_3=None, format_obj_4=None):
```

Comment:
No need to change in this PR, but I'm very much not a fan of these parameter names